docs/tutorials/uky_flowsheet-solution.ipynb

```diff
@@ -1660,6 +1655,14 @@
     "        m.fs.rougher_mixer,\n",
     "    ]\n",
     "\n",
+    "    initializer_sx = SolventExtractionInitializer()\n",
+    "    sx_units = [\n",
+    "        m.fs.solex_rougher_load,\n",
+    "        m.fs.solex_rougher_scrub,\n",
+    "        m.fs.solex_cleaner_load,\n",
+    "        m.fs.solex_cleaner_strip,\n",
```

Comment:
Why is the rougher stripping unit not initialized here?

Reply:
The simple answer is because the rougher stripping unit fails to initialize when using the SolventExtraction Initializer, so it's currently using the BlockTriangularizationInitializer. Why does this particular unit fail to initialize when the other SX units pass? I'm not sure...

Reply:
There is a residual error, let me see

Reply:
Marcus, I checked in my clone, the rougher strip section is being initialized by the SXInitializer in the `uky_flowsheet.py,` can you check doing the same in your branch once? Also in my branch, the notebook file does not have the solvent extraction initializer for intializing the SX units.

Reply:
In the UKy flowsheet merged into main, the rougher strip section is not being initialized by the SXInitializer: https://github.com/prommis/prommis/blob/68036261867fc4e71e7c5ef1f3cba6600714b146/src/prommis/uky/uky_flowsheet.py#L1701. The notebook file in main does not have the SXInitializer, which is what this PR aims to address.

Reply:
It is strange actually, in the normal flowsheet file, the SXInitializer is working and BTInitializer is not working, whereas in pytest file, the reverse is happening

Reply:
Interesting. For me, the SXInitializer isn't working locally in the flowsheet file or the pytest file. I've added a TODO note in the UKy flowsheet about this issue. As I said earlier though, I don't think this is a high-priority concern.

Reply:
Okay.